src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -140,6 +147,68 @@ public String getApiUrl() {
         return apiUrl.toString();
     }
 
+    public String getEmail() throws BayesianException{
+        
+        
+        String url = getOSIOUserUrl();
+        
+        if(url.equals("No-Filter-Found")) {
+        	return "No-Email-Found";
+        }
+        
+        Gson gson;
+        User responseObj;
+
+        HttpGet httpGet = new HttpGet(url);
+        try (CloseableHttpClient client = HttpClients.createDefault();
+                CloseableHttpResponse response = client.execute(httpGet)) {
+
+	            HttpEntity entity = response.getEntity();
+	          	InputStream is = entity.getContent();
```

Comment:
I believe you're mixing tabs and spaces in this method. Please stick with spaces :)

Reply:
I think this InputStream should be closed once we are done with it.